packages/react-components/src/components/orders/OrderList.tsx

```diff
@@ -198,20 +198,21 @@ export function OrderList({
   }, [orders, subscriptions])
   const LoadingComponent = loadingElement || <div>Loading...</div>
   const headerComponent = table.getHeaderGroups().map((headerGroup) => {
-    const columns = headerGroup.headers.map((header, k) => {
+    const columnsComponents = headerGroup.headers.map((header, k) => {
       const sortLabel =
         header.column.getIsSorted() !== false ? header.column.getIsSorted() : ''
       return (
         <ThHtmlElement
           data-testid={`thead-${k}`}
           data-sort={`${sortLabel || ''}`}
           key={header.id}
+          className={columns[k]?.className}
         >
           <span
             {...{
-              className: header.column.getCanSort()
-                ? 'cursor-pointer select-none'
-                : '',
+              className: `${columns[k]?.titleClassName} ${
```

Comment:
```suggestion
              className: `${columns[k]?.titleClassName ?? ''} ${
```
prevent `undefined` from being printed as a string

Reply:
Uh.. Sorry for the oversight.. I fixed it!